mix.exs

```diff
@@ -42,7 +42,8 @@ defmodule GrpcReflection.MixProject do
       {:credo, "~> 1.7", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.4", only: [:dev, :test], runtime: false},
       {:grpc, "~> 0.7"},
-      {:protobuf, "~> 0.11"}
+      {:protobuf, "~> 0.11"},
+      {:google_protos, "~> 0.4.0"}
```

Comment:
`google_protos` is only used in the unit tests and can be marked `only: :test` here

Reply:
`google_protos` is used for unit tests and `example/helloworld`. According to your suggestion, I will mark it `only: :test` and add the dependence to `mix.exs` in `exmaple/helloworld`.

Reply:
I added `only: :test` to `grpc-reflection/mix.exs`, then added `{:google_protos, "~> 0.4.0"}` to `example/helloworld/mix.exs` even with `override: true`, `example/helloworld` failed to run and threw following error.
```
11:57:00.494 [error] ** (CaseClauseError) no case clause matching: {:error, {:undef, [{Google.Protobuf.Timestamp, :descriptor, [], []},
```

Reply:
I see that too, this is really unfortunate.  Our use of the `:google_protos` package for testing is going to force all consuming libraries to also install and load the google protos modules whether they want to or not.